File: Clases2023-24/04-Repaso_Practica13_Retrofit/RetrofitFINAL_github/app/src/main/java/com/example/retrofitfinal/hotel/Price.java
package com.example.retrofitfinal.hotel;

import com.google.gson.annotations.SerializedName;

public class Price {
    @SerializedName("current")
    private String current;
    @SerializedName("exactCurrent")
    private double exactCurrent;

    public Price() {
        //Empty Constructor
    }

    public String getCurrent() {
        return current;
    }

    public void setCurrent(String current) {
        this.current = current;
    }

    public double getExactCurrent() {
        return exactCurrent;
    }

    public void setExactCurrent(double exactCurrent) {
        this.exactCurrent = exactCurrent;
    }
}

File: KOTLIN_MVVM_APP/app/src/main/java/com/tokioschol/travellingkotlin/data/response/Price.kt
package com.tokioschol.travellingkotlin.data.response

import com.google.gson.annotations.SerializedName

data class Price(

	@field:SerializedName("current")
	val current: String? = null,

	@field:SerializedName("old")
	val old: String? = null,

	@field:SerializedName("exactCurrent")
	val exactCurrent: Double? = null
)